src/pages/Home/Home.tsx

```diff
@@ -1,9 +1,30 @@
+import { Box, Button, Container, Typography } from '@mui/material';
 import React from 'react';
+import { main, container, card } from './Home.styles';

```

Comment:
Don't destructure the styles instead use them under the styles object.

```suggestion
import  styles from './Home.styles';
```